django_stubs_ext/tests/test_monkeypatching.py

```diff
@@ -26,15 +27,21 @@ def make_generic_classes(
     request: FixtureRequest,
     monkeypatch: MonkeyPatch,
 ) -> _MakeGenericClasses:
+    _extra_classes: list[type] = []
```

Comment:
`= []` is mutable, this is a big no-no for a default value. Can we please use `= None`?

Reply:
This line isn't a default value, it's for collecting the classes to "unpatch" at the end of the test. I assume this comment was meant for https://github.com/typeddjango/django-stubs/blob/ff9f7bbc9bc98541c8de5d752dd59a4a79610ef0/django_stubs_ext/tests/test_monkeypatching.py#L40 or https://github.com/typeddjango/django-stubs/blob/ff9f7bbc9bc98541c8de5d752dd59a4a79610ef0/django_stubs_ext/django_stubs_ext/patch.py#L61 instead? The mutable default here isn't a huge problem in my opinion because they're typed as `Iterable` and not `list`, so any operation inside the function that would mutate them would cause Mypy to complain anyway. I've changed them now though.